config/database.cr

```diff
@@ -1,10 +1,14 @@
 database = "lucky_migrator_dev"
 
 LuckyRecord::Repo.configure do
-  settings.url = LuckyRecord::PostgresURL.build(
-    hostname: "localhost",
-    database: database
-  )
+  if ENV["DATABASE_URL"]?
+    settings.url = ENV["DATABASE_URL"]
+  else
+    settings.url = LuckyRecord::PostgresURL.build(
+      hostname: "localhost",
+      database: database
+    )
+  end
```

Comment:
This could be simplified to:

```crystal
settings.url = ENV["DATABASE_URL"]? || LuckyRecord::PostgresURL.build(
      hostname: "localhost",
      database: database
    )
```